_layouts/cv.html

```diff
@@ -0,0 +1,119 @@
+---
+layout: default
+---
+<!-- _layouts/cv.html -->
+        <div class="post">
+
+          <header class="post-header">
+            <h1 class="post-title">{{ page.title }} {% if page.cv_pdf %}<a href="{{ page.cv_pdf | prepend: 'assets/pdf/' | relative_url}}" target="_blank" rel="noopener noreferrer" class="float-right"><i class="fas fa-file-pdf"></i></a>{% endif %}</h1>
+            <p class="post-description">{{ page.description }}</p>
+          </header>
+
+          <article>
+            
+            <div class="cv">
+              {% for entry in site.data.cv %}
+                <div class="card mt-3 p-3">
+                  <h3 class="card-title font-weight-medium">{{ entry.title }}</h3>
+                  <div>
+                  {% if entry.type == "list" %}
+                    <ul class="card-text font-weight-light list-group list-group-flush">
+                    {% for content in entry.contents %}
+                      <li class="list-group-item">{{ content}}</li>
+                    {% endfor %}
+                    </ul>
+                  {% elsif entry.type == "map" %}
+                    <table class="table table-sm table-borderless table-responsive">
+                      {% for content in entry.contents %}
+                        <tr>
+                          <td class="p-1 pr-2 font-weight-bold"><b>{{ content.name }}</b></td>
+                          <td class="p-1 pl-2 font-weight-light text">{{ content.value }}</td>
+                        </tr>
+                      {% endfor %}
+                    </table>
+                  {% elsif entry.type == "nested_list" %}
```

Comment:
quick question: do you think it's possible to implement nested list (or any nested structure) using [recursion](https://jekyllrb.com/tutorials/navigation/#scenario-9-nested-tree-navigation-with-recursion)?

Reply:
I tried implementing this, but was unsuccessful, each list sub item has different classes and formatting. May be, to implement this we may have to change the style a bit.

Reply:
thank you for giving it a shot! let's leave it as a future improvement.